gui/default/index.html

```diff
@@ -880,6 +880,7 @@ <h4 class="panel-title">
   <script type="text/javascript" src="syncthing/core/notificationDirective.js"></script>
   <script type="text/javascript" src="syncthing/core/pathIsSubDirDirective.js"></script>
   <script type="text/javascript" src="syncthing/core/popoverDirective.js"></script>
+  <script type="text/javascript" src="syncthing/core/pushNotificationsService.js"></script>
```

Comment:
Hi, where can I find this file?

Reply:
Oh sorry, it seems I haven't committed that one. Pushing it soon